plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -188,6 +193,44 @@ def __cmp__(self, other):
             other.complete_version_serialized
         )
 
+    def get_or_calculate_checksum(self, checksumtype):
+        """
+        Tries to get the sha1 or sha256 checksum value from the attribute on self of the
```

Comment:
This docblock paragraph needs revision to match the generalized implementation. I think this was leftover from before.